generated/nidaqmx/_task_modules/channels/ci_channel.py

```diff
@@ -8111,7 +8111,7 @@ def ci_pulse_freq_logic_lvl_behavior(self):
         check_for_error(error_code)
 
     @property
-    def ci_pulse_freq_starting_edge(self):
+    def ci_pulse_freq_start_edge(self):
```

Comment:
I think I'm ok with `start` over `starting`. That is what the C API uses, and it still reads fine. But lets add deprecations, like [ai](https://github.com/ni/nidaqmx-python/blob/master/generated/nidaqmx/_task_modules/channels/ai_channel.py#L9379).

Reply:
Added deprecation for the `ci_pulse_freq_starting_edge`